app/assets/javascripts/omni_nav/google-custom-search.js

```diff
@@ -0,0 +1,314 @@
+var GoogleCustomSearch = (function() {
```

Comment:
I've broken this out of the main `OmniNav2` module. Here's the before:

https://github.com/chapmanu/cascade-assets/blob/2758afd76755884feb074b9abdafbe394ad05b93/app/assets/javascripts/omni-nav-v2.js#L325